reports/glides_terminal_departure_accuracy.livemd

```diff
@@ -111,71 +111,67 @@ use_advanced_date_input =
 
 frame = Kino.Frame.new(placeholder: false)
 
+content_map = %{
+  true:
+    Kino.Layout.grid(
+      [
+        env_input,
+        Kino.Layout.grid(
+          [
+            start_date_input,
+            end_date_input,
+            sample_rate_input,
+            samples_per_minute_input
+          ],
+          columns: 2
+        )
+      ],
+      columns: 1
+    ),
+  false:
+    Kino.Layout.grid(
+      [
+        env_input,
+        date_input,
+        sample_rate_input,
+        samples_per_minute_input
+      ],
+      columns: 2
+    )
+}
+
+doc_table_map =
+  %{
+    true: """
+    | Analyze data from... | Start analyzing data from this date/time.<br/>Default is 4am yesterday, Eastern. |
+    | to...                | Analyze data up to this date/time.<br/>Default is 3:59am today, Eastern.         |
+    """,
+    false: """
+    | Date | Service date to analyze data from. A 24-hour period starting at 4am Eastern. |
+    """
+  }
+  |> Map.new(fn {k, fragment} ->
+    {k,
+     Kino.Markdown.new("""
+     ### Setting Details
+
+     | Setting                             | Details                                                                                                                                                                                              |
+     | ----------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+     | Environment                         | Environment to analyze data from.                                                                                                                                                                    |
+     #{String.trim_trailing(fragment)}
+     | Sample data at (?)-minute intervals | Sets interval at which data is sampled for analysis.<br/>Lower value = more samples and slower report generation.                                                                                    |
+     | Take (?) samples per minute         | Sets number of samples to take within each sampled minute.<br/>Higher value = more samples and slower report generation.<br/>Leave blank to analyze *all* data within each sampled minute. (slowest) |
+     """)}
+  end)
+
 update_frame = fn
   nil ->
     Kino.Frame.clear(frame)
 
   use_advanced_date ->
```

Comment:
I hit some intermittent issues with re-evaluation and the  "use advanced date" dropdown -- the first time, it helpfully prompted me to choose Yes or No, but on subsequent re-evaluations, I often received a "input not found" error message from Kino while trying to fetch the data.

I hit this while updating my AWS credentials but I can reproduce it without running that cell.

![image](https://github.com/user-attachments/assets/43f70452-a79c-45c6-9b2d-50805ccee678)